app/src/main/java/org/wikipedia/util/DeviceUtil.java

```diff
@@ -111,6 +111,12 @@ public static void updateStatusBarTheme(@NonNull Activity activity, @Nullable To
         }
     }
 
+    public static void setRightClickEvent(@NonNull View view) {
```

Comment:
This name could be much more specific. Something like `setContextClickAsLongClick`.

Reply:
Done!